README.md

```diff
@@ -18,36 +18,59 @@ To access the serverinfo API you will need the credentials of an admin user. It
 
 ```plain
 $ nextcloud-exporter --help
-Usage of nextcloud-exporter:
-  -a, --addr string        Address to listen on for connections. (default ":9205")
-  -p, --password string    Password for connecting to nextcloud.
-  -t, --timeout duration   Timeout for getting server info document. (default 5s)
-  -l, --url string         URL to nextcloud serverinfo page.
-  -u, --username string    Username for connecting to nextcloud.
+  Usage of nextcloud-exporter:
+    -a, --addr string          Address to listen on for connections. (default ":9205")
+    -c, --config-file string   Path to YAML configuration file.
+    -p, --password string      Password for connecting to Nextcloud.
+    -t, --timeout duration     Timeout for getting server info document. (default 5s)
+    -l, --url string           URL to Nextcloud serverinfo page.
+    -u, --username string      Username for connecting to Nextcloud.
 ```
 
-Some settings can also be specified through environment variables:
+After starting the server will offer the metrics on the `/metrics` endpoint, which can be used as a target for prometheus.
 
-Name                     | Description
--------------------------|-------------------------------------
-NEXTCLOUD_SERVERINFO_URL | URL to nextcloud serverinfo page
-NEXTCLOUD_USERNAME       | Username for connecting to nextcloud
-NEXTCLOUD_PASSWORD       | Password for connecting to nextcloud
+### Configuration methods
 
-Command line arguments take precedence over environment variables.
+There are three methods of configuring the nextcloud-exporter (higher methods take precedence over lower ones):
 
-After starting the server will offer the metrics on the `/metrics` endpoint, which can be used as a target for prometheus.
+- Environment variables
+- Configuration file
+- Command-line parameters
 
-The exporter will query the nextcloud server every time it is scraped by prometheus. If you want to reduce load on the nextcloud server you need to change the scrape interval accordingly:
+#### Environment variables
 
-```yml
-scrape_configs:
-  - job_name: 'nextcloud'
-    scrape_interval: 90s
-    static_configs:
-      - targets: ['localhost:9205']
+All settings can also be specified through environment variables:
+
+    Environment variable | Flag equivalent
+-----------------------: | :--------------
+NEXTCLOUD_LISTEN_ADDRESS | --addr
+NEXTCLOUD_PASSWORD       | --password
+NEXTCLOUD_TIMEOUT        | --timeout 
+NEXTCLOUD_SERVERINFO_URL | --url
+NEXTCLOUD_USERNAME       | --username
```

Comment:
I just saw, that the formatting here is broken in the [rendered version](https://github.com/WilliButz/nextcloud-exporter/blob/ef7f87456086a994d86c5dcc9bcc727185ce3050/README.md), prefixing every line with another pipe and space seems to do the trick :)